tests/test_offchain_client.py

```diff
@@ -23,3 +23,24 @@ def process_inbound_request(x_request_id: str, jws_key_address: str, content: by
     command = offchain.PaymentCommand(payment=payment, my_actor_address=payment.sender.address, inbound=False)
     resp = sender_client.send_command(command, sender.compliance_key.sign)
     assert resp
+
```

Comment:
Add a param fixture, it generates command by params defined in fixture (see https://docs.pytest.org/en/stable/fixture.html#parametrizing-fixtures)
```
@pytest.fixture(
    scope="module",
    params=[offchain.CommandType.PaymentCommand, offchain.CommandType.Xxxxx]
)
def command_type(request):
    return request.param
```

then change test_send_and_deserialize_request to accept command_type as argument,

```
command = factory.new_command(command_type, sender, receiver)
resp = sender_client.send_command(command, sender.compliance_key.sign)
```

add new_command with command_type to factory

Reply:
COOL! done